commons-statistics-descriptive/src/main/java/org/apache/commons/statistics/descriptive/Sum.java

```diff
@@ -0,0 +1,138 @@
+ *
+ * <p>This class is designed to work with (though does not require)
+ * {@linkplain java.util.stream streams}.
+ *
+ * <p><strong>This implementation is not thread safe.</strong>
+ * If multiple threads access an instance of this class concurrently,
+ * and at least one of the threads invokes the <code>accept()</code> or
+ * <code>combine()</code> method, it must be synchronized externally.
+ *
+ * <p>However, it is safe to use <code>accept()</code> and <code>combine()</code>
+ * as <code>accumulator</code> and <code>combiner</code> functions of
+ * {@link java.util.stream.Collector Collector} on a parallel stream,
+ * because the parallel implementation of {@link java.util.stream.Stream#collect Stream.collect()}
+ * provides the necessary partitioning, isolation, and merging of results for
+ * safe and efficient parallel execution.
+ *
+ * @since 1.1
+ */
+public abstract class Sum implements DoubleStatistic, DoubleStatisticAccumulator<Sum> {
+
+    /**
+     * Create a Sum instance.
+     */
+    Sum() {
+        //No-op
+    }
+
+    /**
+     * Creates a {@code Sum} implementation which does not store the input value(s) it consumes.
+     *
+     * <p>The result is <code>NaN</code> if any of the values is <code>NaN</code> or the sum
+     * at any point is a <code>NaN</code>.
+     *
+     * <p>The result is zero if no values have been added.
+     *
+     * <p>Uses a {@link org.apache.commons.numbers.core.Sum Sum} delegate under the hood
+     * to update and compute the {@code sum}.
+     *
+     * @return {@code Sum} implementation.
+     */
+    public static Sum create() {
+        return new WrappedSum();
+    }
+
+    /**
+     * Returns a {@code Sum} instance that has the sum of all input value(s).
+     *
+     * <p>The result is <code>NaN</code> if any of the values is <code>NaN</code>
+     * or the sum at any point is a <code>NaN</code>.
+     *
+     * <p>When the input is an empty array, the result is zero.
+     *
+     * @param values Values.
+     * @return {@code Sum} instance.
+     */
+    public static Sum of(double... values) {
+        final WrappedSum sum = new WrappedSum();
+        Arrays.stream(values).forEach(sum);
+        return sum;
+    }
+
+    /**
+     * Updates the state of the statistic to reflect the addition of {@code value}.
+     * @param value Value.
+     */
+    @Override
+    public abstract void accept(double value);
+
+    /**
+     * Gets the sum of all input values.
+     *
+     * <p>When no values have been added, the result is zero.
+     *
+     * @return {@code Sum} of all values seen so far.
+     */
+    @Override
+    public abstract double getAsDouble();
+
+    /** {@inheritDoc} */
+    @Override
+    public abstract Sum combine(Sum other);
+
+    /**
+     * {@code Sum} implementation that does not store the input value(s) processed so far.
+     *
+     * <p>Uses a {@link org.apache.commons.numbers.core.Sum Sum} delegate under the hood
```

Comment:
Simplify to `Delegates to the {@link org.apache.commons.numbers.core.Sum} implementation`.